client/src/SignupNonProfit.tsx

```diff
@@ -59,129 +101,184 @@ function SignupNonProfit() {
+                                        id="last_name"
+                                        label="Last Name"
+                                        placeholder="Last Name"
+                                        value={formData.last_name}
+                                        onChange={handleFieldChange}
+                                    />
+                                </Grid>
+                                <Grid item md={12} xs={12}>
+                                    <TextField
+                                        id="role_or_title"
+                                        label="Role Title"
+                                        placeholder="Role Title"
+                                        value={formData.role_or_title}
+                                        onChange={handleFieldChange}
+                                    />
+                                </Grid>
+                                <Grid item md={6} xs={12}>
+                                    <TextField
+                                        id="email"
+                                        label="Email"
+                                        placeholder="Email"
+                                        value={formData.email}
+                                        onChange={handleFieldChange}
+                                    />                            
+                                </Grid>
+                                <Grid item md={6} xs={12}>
+                                    <TextField
+                                        id="password"
+                                        label="Password"
+                                        placeholder="Password"
+                                        value={formData.password}
+                                        onChange={handleFieldChange}
+                                    />                            
+                                </Grid>
+                                    <FormControlLabel
+                                        style={{ textAlign: 'left', display: 'block' }}
+                                        control={
+                                            <Checkbox
+                                                color="primary"
+                                                checked={formData.accept_terms}
+                                                onChange={handleFieldChange}
+                                                name="accept_terms"
+                                                inputProps={{ 'aria-label': 'accept_terms_checkbox' }}
+                                            />
+                                        }
+                                        label={
+                                            <label>
+                                                Accept the{' '}
+                                                <StyledLink to="/terms_and_condtions" target="_blank">
```

Comment:
`to="/terms_of_service"` maybe?
that's a placeholder we've added already, might be what we're looking for here 🤷